### PR TITLE
Verification job tag search Types and Documentation

### PR DESCRIPTION
**Pull request by gdsbrooks**
### Changes Made
- Created annoated public types for Verification Job Tag Search
- added to Postman Documentation
- p2o auto generated swagger.yml

WITH: https://github.com/ClicknClear/cnc-verification/pull/640
### Checklist
- [x] Self-Review - have you reviewed your code prior to raising this as a pull request.

### CODE REVIEW

File: packages/lvs-types/src/v1/verificationJob.ts [hunk 1]
```diff
@@ -123,3 +123,23 @@ export interface IVerificationJobStatusUpdateV1 {
...
+ * @param tags An array of strings, Required. At least one tag must be included for the search to yield results.
+ * @param statuses An array of VerificationJobStatus[]. Optional. It narrows the search to jobs in the specified status(s).
+ * @param limit Pagination size limit. Optional. Defaults to 25.
+ * @param offset Paginiation offset. Optional. If specified, 'limit' must also be passed.
+ */
+export type VerificationJobTagSearchV1 = z.infer<typeof verificationJobTagSearchSchemaV1>
+export const verificationJobTagSearchSchemaV1 = z.object({
+  tags: z.string().array().min(1),
+  statuses: z.nativeEnum(VerificationJobStatusV1).array().optional(),
+  limit: z.number().default(25).optional(),
```
[the remark this answers is not in the document]
Comment (gdsbrooks) [reply packages/lvs-types/src/v1/verificationJob.ts:138]: Resolved by: 73803112ea8eca941e5933c8998d1a0221ea9a52

File: docs/apis/swagger-docs.yml [hunk 2]
```diff
@@ -1266,3 +1266,185 @@ paths:
                               downloadURL: >-
                                 http://lvs.local.clicknclear.com:3001/api/files/download?t=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJidWNrZXQiOiJjbGlja25jbGVhci12ZXJpZmljYXRpb24tbGljZW5zZXMtc3RhZ2luZyIsImtleSI6ImV4dGVybmFsLWxpY2Vuc2VzL2MxOTg2MjBlZGYyMmI5YzM3ZDRiNjliNjMyOWY1NDNlLnBkZiIsImZpbGVuYW1lIjoiVW5rbm93bi5wZGYiLCJjb250ZW50VHlwZSI6ImFwcGxpY2F0aW9uL3BkZiIsImlhdCI6MTcyNjU1ODMwNSwiZXhwIjoxNzI2NjQ0NzA1fQ.sJXEJbXqDSXXZheAxRKSts9WaywT3A-VlT5HlEO9aVY
                         publicDomainProof: null
+  /api/public/v1/verification-job/search:
+    post:
+      tags:
+        - Verification Job V1
+      summary: Search Verification Jobs
+      description: >-
```
[the remark this answers is not in the document]
Comment (gdsbrooks) [reply docs/apis/swagger-docs.yml:1274]: Added names in the relevant paragraph.
Resolved by: 73803112ea8eca941e5933c8998d1a0221ea9a52

File: packages/lvs-types/src/v1/verificationJob.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (TristanBarlow) [reply packages/lvs-types/src/v1/verificationJob.ts:138]: Oh sorry I meant on the limit, currently they could request 1m jobs and potentially cause an issue with the server! 
But yeah max on tags would be good too!

[the remark this answers is not in the document]
Comment (gdsbrooks) [reply packages/lvs-types/src/v1/verificationJob.ts:138]: hahah gotcha

[the remark this answers is not in the document]
Comment (gdsbrooks) [reply packages/lvs-types/src/v1/verificationJob.ts]: Resolved by: 0f9329596e111dd2c50b5d074b44c2ab751d713d

**Review by TristanBarlow — APPROVED** [review]

**Pull request merged**